### PR TITLE
feat: display "LINKED ACCOUNT" badge and card spacing

### PR DESCRIPTION
**Pull request by peterpeterparker**
# Motivation

Linked account should be displayed with a badge "LINKED ACCOUNT". In the nns-dapp rewrite, this information will not be displayed at the end of the card but next after the account title.

In addition, this PR also fix the display of long account name and neurons card on small devices.

Finally, texts `[community fund]` and `[hotkey control]` modified to capital case without `[]`.

All the ui changes have been reviewed by Mischa.

# Changes

- add and use `AccountBadge`
- mixin for cards title that use `start` and `end` slot
- use above mixin in neuron, proposal and account cards
- `[community fund]` and `[hotkey control]` to `COMMUN...` and `HOTK...`

# Screenshots

Capture d’écran 2022-03-29 à 15 31 54
Capture d’écran 2022-03-29 à 15 31 59
Capture d’écran 2022-03-29 à 15 32 03
Capture d’écran 2022-03-29 à 15 32 16
Capture d’écran 2022-03-29 à 15 32 30
Capture d’écran 2022-03-29 à 15 32 34
Capture d’écran 2022-03-29 à 15 32 41

### CODE REVIEW

File: frontend/svelte/src/tests/lib/components/accounts/AccountBadge.spec.ts [hunk 1]
```diff
@@ -0,0 +1,30 @@
...
+    });
+
+    expect(() =>
+      getByText(en.accounts.linked_account, { exact: false })
+    ).toThrow();
+  });
+
+  it("should render linked account badge", () => {
+    const { getByText } = render(AccountBadge, {
+      props: { account: { ...mockMainAccount, subAccount: ["test"] } },
```
Comment (lmuntanerDfinity) [inline frontend/svelte/src/tests/lib/components/accounts/AccountBadge.spec.ts:23]: Isn't subAccount an array of numbers? How come TS doesn't fail?

File: frontend/svelte/src/lib/components/accounts/AccountBadge.svelte [hunk 2]
```diff
@@ -0,0 +1,12 @@
...
+  import type { Account } from "../../types/account";
+  import { i18n } from "../../stores/i18n";
+
+  export let account: Account;
+</script>
+
+<!-- No badge for main account -->
+
+{#if account.subAccount !== undefined}
+  <span><small>{$i18n.accounts.linked_account}</small></span>
```
Comment (lmuntanerDfinity) [inline frontend/svelte/src/lib/components/accounts/AccountBadge.svelte:11]: do we need the `<span>`?

**Review by lmuntanerDfinity — COMMENTED** [review]
A few comments

File: frontend/svelte/src/tests/lib/components/accounts/AccountBadge.spec.ts (hunk 1 above)
Comment (peterpeterparker) [reply frontend/svelte/src/tests/lib/components/accounts/AccountBadge.spec.ts]: it is not typed here. modifid to array of a number

Comment (lmuntanerDfinity) [reply frontend/svelte/src/tests/lib/components/accounts/AccountBadge.spec.ts]: ok, good to know.

File: frontend/svelte/src/lib/components/accounts/AccountBadge.svelte (hunk 2 above)
Comment (lmuntanerDfinity) [reply frontend/svelte/src/lib/components/accounts/AccountBadge.svelte]: I didn't know that, feel free to add the `span` back, I was just wondering about it. Not asking for the change.

Comment (peterpeterparker) [reply frontend/svelte/src/lib/components/accounts/AccountBadge.svelte]: It's conform without too no worries

**Review by lmuntanerDfinity — APPROVED** [review]

**Pull request merged**